src/Widget/BaseWidget.php

```diff
@@ -20,7 +20,7 @@ class BaseWidget implements WidgetInterface
     protected $priority = 0;
 
     /** @var string */
-    protected $template;
+    protected $template = '';
```

Comment:
why?

Reply:
Because you requested the change to `$this->getTemplate()`, and it has `: string`. Even when a widget has no template, it still gets called, and this prevents it from breaking.

Reply:
If it doesn't have template then maybe it shouldn't extend BaseWidget